docs/app-rails/technical-foundation.md

```diff
@@ -61,7 +60,7 @@ To preview email views in the browser, visit: `/rails/mailers`
 
 To test AWS SES email sending locally:
 
-1. Set the `AWS_*` env var in your `.env` file
+1. Set the "AWS services" env var in your `.env` file
```

Comment:
env vars?
env variables?
Since it is multiple vars, it's something someone might get tripped up on if they see it singular here.

If aws bucket (now just bucket) is moved out of AWS services section, is it needed here for SES to work properly?